### ISSUE TITLE
Publish Linux ARM builds

### ISSUE THREAD

**Issue by c-w** [CloudCannon/pagefind#147]
On a Macbook M1 running Docker, pagefind fails to install on the official Node LTS base image:

```sh
docker run --rm node:lts npm install --global pagefind@0.9.3
```

Here's the stacktrace:

```
#8 19.64 Directory: /node_modules/pagefind
#8 19.64 Output:
#8 19.64 Finding release for v0.9.3
#8 19.64 https://api.github.com/repos/CloudCannon/pagefind/releases/tags/v0.9.3
#8 19.64 GET https://api.github.com/repos/CloudCannon/pagefind/releases/tags/v0.9.3
#8 19.64 Deleting invalid download cache
#8 19.64 Downloading pagefind failed: Error: Asset not found with name: pagefind_extended-v0.9.3-aarch64-unknown-linux-gnu.tar.gz
#8 19.64     at getAssetFromGithubApi (/node_modules/pagefind/lib/download.js:192:15)
#8 19.64     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
#8 19.64     at async module.exports (/node_modules/pagefind/lib/download.js:290:9)
#8 19.64     at async main (/node_modules/pagefind/lib/postinstall.js:71:9)
```

If I add `musl-tools` and `musl-dev` to the base image and download a specific `x86_64` release, it works:

```sh
docker run --rm node:lts sh -c '
apt-get -y update &&
apt-get -y install musl-tools musl-dev &&
curl -sfLO https://github.com/CloudCannon/pagefind/releases/download/v0.10.0/pagefind_extended-v0.10.0-x86_64-unknown-linux-musl.tar.gz &&
tar xf pagefind_extended-v0.10.0-x86_64-unknown-linux-musl.tar.gz &&
echo "<html><h1>hi</h1></html>" > /tmp/foo.html &&
./pagefind_extended --source /tmp'
```

However, this is quite clunky so it would be ideal if the installer could be fixed on this setup.

**Comment by bglw** [comment]
I'll need to get the CI to build/release some Linux/arm builds for this — though curious that the `x86_64` build runs for you on arm linux, I wonder if that's universal or some Docker multi-platform magic 🤔 I'll have to test that.

Most likely the `musl-tools musl-dev` install will continue to be required. Pagefind currently only builds a `musl` target for Linux, which is supported on almost any `x64` architecture. If building for arm, it'll be an `aarch64-unknown-linux-musl` rather than a `aarch64-unknown-linux-gnu`, which looks like it requires the `musl` support to be specifically installed. (I could look into building a `gnu` binary, but it was dodged for a reason 😅)

**Comment by bglw** [comment]
👋 Hi @c-w — a Linux ARM build is now included in [Pagefind v0.10.2](https://github.com/CloudCannon/pagefind/releases/tag/v0.10.2) 🎉 

I haven't found installing `musl-tools` or `musl-dev` to be a requirement, let me know if that isn't the case.

**Comment by c-w** [comment]
Confirmed the fix, thanks so much! 🙌
